### ISSUE TITLE
Adding Angular 12 Support

### ISSUE THREAD

**Issue by cwoebker** [orizens/ngx-infinite-scroll#382]
Your issue may already be reported!
Please search on the [issue track](../) before creating one.

## Expected Behavior

It would be awesome if Angular 12 support would be added. Specifically supporting the Angular 12 Material library.

## Actual Behavior

When running an Angular 12 project with the ngx-infinite-scroll dependency, I am getting the following errors while building the project:
```
./node_modules/ng-mat-select-infinite-scroll/fesm2015/ng-mat-select-infinite-scroll.js:17:34-55 - Error: export 'SELECT_ITEM_HEIGHT_EM' (imported as 'SELECT_ITEM_HEIGHT_EM') was not found in '@angular/material/select' (possible exports: MAT_SELECT_CONFIG, MAT_SELECT_SCROLL_STRATEGY, MAT_SELECT_SCROLL_STRATEGY_PROVIDER, MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY, MAT_SELECT_TRIGGER, MatSelect, MatSelectChange, MatSelectModule, MatSelectTrigger, _MatSelectBase, matSelectAnimations)

./node_modules/ng-mat-select-infinite-scroll/fesm2015/ng-mat-select-infinite-scroll.js:66:67-88 - Error: export 'SELECT_ITEM_HEIGHT_EM' (imported as 'SELECT_ITEM_HEIGHT_EM') was not found in '@angular/material/select' (possible exports: MAT_SELECT_CONFIG, MAT_SELECT_SCROLL_STRATEGY, MAT_SELECT_SCROLL_STRATEGY_PROVIDER, MAT_SELECT_SCROLL_STRATEGY_PROVIDER_FACTORY, MAT_SELECT_TRIGGER, MatSelect, MatSelectChange, MatSelectModule, MatSelectTrigger, _MatSelectBase, matSelectAnimations)
```

## Possible Solution

Removing or replacing the dependency on the `SELECT_ITEM_HEIGHT_EM` variable of the material select component.

### Steps To Reproduce / Demo (if applicable)
[ ] Please share a [stackblitz](https://stackblitz.com/) or [plunkr](https://plnkr.co/) demo reproducing this issue  

### Your Environment

- [x] Version used: Angular 12 Release
- [x] Browser Name and version: N/A
- [x] (Optional) Operating System and version (desktop or mobile): N/A
- [x] (Optional) Link to your project: N/A

**Comment by cwoebker** [comment]
@orizens Hi Oren, are you open to accepting pull requests to get the Angular 12 support going? I might be able to get in some resources here.

**Comment by orizens** [comment]
@cwoebker sure. please do.

**Comment by cwoebker** [comment]
@orizens Hi, so I just took a first look and realized that I reported an issue with a different library here. The error was related to this project: https://github.com/HaidarZ/ng-mat-select-infinite-scroll and it was fixed in their last 3.0.0 release. Sorry for the confusion, I will close this issue accordingly.
